fda-model/metrics/ImpactAreaScenarioResults.cs

```diff
@@ -168,10 +171,16 @@ public bool ResultsAreConverged(double upperConfidenceLimitProb, double lowerCon
                 if (computeWithDamage == true)
                 {
                     foreach (ConsequenceDistributionResult consequenceDistributionResult in ConsequenceResults.ConsequenceResultList)
-                    {
-                        if (consequenceDistributionResult.ConsequenceHistogram.IsHistogramConverged(upperConfidenceLimitProb, lowerConfidenceLimitProb) == false)
+                    {   
+                        if(consequenceDistributionResult.ConsequenceHistogram.HistogramIsZeroValued)
+                        {
+                            eadIsConverged = true;
+                        } else
                         {
-                            eadIsConverged = false;
+                            if (consequenceDistributionResult.ConsequenceHistogram.IsHistogramConverged(upperConfidenceLimitProb, lowerConfidenceLimitProb) == false)
```

Comment:
This is fine, and is maybe easier to read, but often if you are doing a check and then the only thing you are doing in the check is assigning a boolean value, you can just do that in one line:

eadIsConverged = consequenceDistributionResult.ConsequenceHistogram.IsHistogramConverged(upperConfidenceLimitProb, lowerConfidenceLimitProb);